pkg/apis/kudo/v1beta1/parameter_types.go

```diff
@@ -30,4 +30,7 @@ type Parameter struct {
 
 	// Specifies if the parameter can be changed after the initial installation of the operator
 	Immutable *bool `json:"immutable,omitempty"`
+
+	// Defines a list of allowed values. If Default is set and Enum is not nil, the value must be in this list as well
+	Enum *[]string `json:"enum,omitempty"`
```

Comment:
The intention was to make it clear if any value is set or not - but as an "enum" with no entries would be invalid anyway, it could be an unpointered slice as well. 

I kind of like it as a pointer, but i'm not really married to it - if it bothers you, i'll change it :)

Reply:
ah you mean you want to validate for user providing 
```
enum:
default: xxx
```

? Well ... okay :)